### ISSUE TITLE
Redesign /Standards

### ISSUE THREAD

**Issue by kerryalyons** [DFE-Digital/accessibility-manual#134]
### Actions

- link to Standards manual open in new tab
- Accessibility conformance standard to be listed out (currently missing)
- Create assurance section at bottom of /Standards - will contain Standard 5 [link], Accessibility specialist role [link], Audits [link]

### Insight 
Only 1/5 users navigated to the conformance content straightforwardly, with most expecting it to be within the standards part of the manual. 

### Insight 
The look & feel of the manual felt familiar to users, matching other DfE products. However, users often didn't notice when they entered the standards manual after leaving the accessibility manual & struggled to navigate back. 

### Screenshot of proposed new navigation

![Image](https://github.com/user-attachments/assets/e1d7ccf0-3e9d-4744-9655-da97307a4bef)

[Older sketches and ideations](https://lucid.app/lucidspark/0152507d-f1ac-4fab-96a8-6b1533789c79/edit?view_items=BhJb87VWB7zn&invitationId=inv_389f520c-af1b-4c91-ad7d-954d9c92c609)

**Comment by kerryalyons** [comment]
New page created for details for laws and regs.

[Content for new page: laws and regulations](https://educationgovuk.sharepoint.com/:w:/r/sites/DesignOps/_layouts/15/Doc.aspx?sourcedoc=%7BD6733BCE-9FE7-4350-86F5-A67EECDCDA66%7D&file=Laws%20and%20regulations.docx&action=default&mobileredirect=true)

This may need it's own issue but storing here for now.
